hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/AbstractOmBucketReadWriteOps.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.hadoop.ozone.OzoneConsts;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.unit.DataSize;
```

Comment:
Ozone already has `StorageSize` in `hdds-config` and `SizeInBytes` from Ratis.  Can we use one of those instead of Spring?

Reply:
Thank you @adoroszlai for the review. This is a good idea instead of relying on the additional Spring dependencies. 

I will be working on making the required changes for utilizing `StorageSize` and `SizeInBytes` instead.